packages/office-ui-fabric-react/src/components/Divider/VerticalDivider.styles.ts

```diff
@@ -0,0 +1,36 @@
+import { IVerticalDividerPropsStyles, IVerticalDividerStyles } from './VerticalDivider.types';
+import { IStyleFunction } from '../../Utilities';
+
+export const getStyles: IStyleFunction<IVerticalDividerPropsStyles, IVerticalDividerStyles> = (
+  props: IVerticalDividerPropsStyles
+): IVerticalDividerStyles => {
+  const { theme, getClassNames } = props;
+
+  if (!theme) {
```

Comment:
Is there a case where we don't have the theme now that we are wrapping it with `styled`?

Reply:
Unsure, but I see that other components check.

Reply:
To check the theme maybe better use this approach:
https://github.com/OfficeDev/office-ui-fabric-react/blob/dc8f7bcba448f00378905026f4a74f1187882a1d/packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.styles.ts#L68-L70
... don't forget to replace the Dropdown word 😄 in the error you throw so that if it happens we know where to look.